packages/fuels-signers/src/lib.rs

```diff
@@ -145,179 +138,4 @@ mod tests {
         Ok(())
         // ANCHOR_END: sign_tx
     }
-
-    #[tokio::test]
-    async fn test_wallet_get_coins() -> fuels_types::errors::Result<()> {
-        const AMOUNT: u64 = 1000;
-        const NUM_COINS: u64 = 3;
-        let mut wallet = WalletUnlocked::new_random(None);
```

Comment:
What happened to these tests? Couldn't find where they were moved. Are they redundant?

Reply:
My bad. Managed to look at a stale version.